src/Titon/Console/Command.hh

```diff
@@ -0,0 +1,101 @@
+<?hh // strict
+/**
+ * @copyright   2010-2015, The Titon Project
+ * @license     http://opensource.org/licenses/bsd-license.php
+ * @link        http://titon.io
+ */
+
+namespace Titon\Console;
+
+use Titon\Console\Input;
+use Titon\Console\Output;
+use Titon\Console\InputDefinition\Argument;
+use Titon\Console\InputDefinition\Flag;
+use Titon\Console\InputDefinition\Option;
+
+/**
+ * A `Command` is a user-defined class that can to register user input and run
+ * using the input provided.
+ *
+ * @package Titon\Console
+ */
+interface Command {
+
+    /**
+     * The bootstrap method that sets up name, description, and necessary parameters
+     * for the `Command` to run.
+     *
+     * @return void
+     */
+    public function configure(): void;
```

Comment:
It can, but this function should really only be called internally anyway. Thoughts?